docs/concepts.md

```diff
@@ -320,7 +322,7 @@ For example, Wordpress requires a MySQL database which can be satisfied by Cloud
 cloud-specific resource classes would be created for CloudSQL, RDS, and Azure DB.
 
 ### Non-Portable Resource Class
-See [cloud-specific resource class](#cloud-specific-resource-class).
+Another term for [cloud-specific resource class](#cloud-specific-resource-class).
```

Comment:
Nitpick: In general I prefer to restate the title, in case the reader is not paying close attention to the content in the titles. So in this case, I would normally use something more like `Non-portable resource class is another term for`. Another option would be to have the terms adjacent to the definitions, like
```
**Non-portable resource class**: another term for . . .
```